sdk/iot/core/inc/az_iot_hub_client.h

```diff
@@ -69,6 +69,10 @@ AZ_NODISCARD az_result az_iot_hub_client_init(
 
 /**
  * @brief Gets the MQTT user name.
+ * 
+ * The user name will be of the following format:
+ * [Format without module id] {iothubhostname}/{device_id}/?api-version=2018-06-30

```

Comment:
>-version=2018-06-30 [](start = 63, length = 19)

If user specifies user_agent in the options, isn't this appended here too?

Reply:
Yes the user params get appended after the api-version. I can put a note after so that user params are accounted for?
```suggestion
 * [Format without module id] {iothubhostname}/{device_id}/?api-version=2018-06-30&{user_params}

```

Reply:
+1

---
In reply to: [393369684](https://github.com/Azure/azure-sdk-for-c/pull/455#discussion_r393369684) [](ancestors = 393369684)

Reply:
updated